Procedure/Procedure-GenomicStudy-Example.json

```diff
@@ -0,0 +1,35 @@
+{
+ "resourceType": "Procedure",
+ "id": "Procedure-GenomicStudy-Example",
+ "extension": [
+  {
+   "url": "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study-analysis-ext",
+   "valueReference": {
+    "reference": "Procedure/Procedure-PGXGenomicStudyAnalysis-Example",
+    "display": "PGX Genomic Study Analysis"
+   }
+  }
+ ],
+ "status": "completed",
+ "category": {
+  "coding": [
+   {
+    "system": "http://snomed.info/sct",
+    "code": "405824009",
```

Comment:
The Genomic Study profile has fixed values here: http://terminology.hl7.org/CodeSystem/observation-category and laboratory